.storybook/config.js

```diff
@@ -1,4 +1,4 @@
-import 'storybook-chromatic';
+import 'chromatic';
```

Comment:
actually, we should be able to delete this one entirely, we don't run chromatic on this storybook